types/index.ts

```diff
@@ -2,6 +2,8 @@ import { Request } from "express";
 import core from "express-serve-static-core";
 import { IncomingHttpHeaders } from "node:http";
 
+export const JQ_HEADER = "braekhus-jq-response-filter";
```

Comment:
This is a jq program, not a filter, right? Were it a filter I presume your invocation would look more like `jq.run(". | " + filter, ...)`.

Should this be
```suggestion
export const JQ_HEADER = "x-braekhus-response-jq-program";
```
?

Reply:
Called it `braekhus-response-jq-transforrm`